ScoutSuite/output/data/inc-scoutsuite/scoutsuite.js

```diff
@@ -97,6 +98,12 @@ function onPageLoad() {
 
 };
 
+function registerHandlebarHelpers() {
+    Handlebars.registerHelper('concat', function(str1, str2) {
+        return [str1, str2].join('');
+    });
+}
+
```

Comment:
There is already a bunch of helpers registered, and I'm pretty sure there is one for concatenation... why did you add `registerHandlebarHelpers`? 

Lemme find that helper

Reply:
I didn't notice the existing ones but the current `concat` joins the string with a dot (ie. `concat str1 str2` returns `str1.str2` instead of `str1str2`). I'll move mine to the `helpers.js` file and rename it.